base_layer/wallet/src/ffi.rs

```diff
@@ -355,10 +432,24 @@ pub unsafe extern "C" fn generate_key_manager_from_seed_words(
     branch_seed: *const c_char, // String
 ) -> bool
 {
+    return true;
 }
 
 #[no_mangle]
-pub unsafe extern "C" fn generate_identity(wallet: *mut WalletFfi) -> *mut IdentityFfi {}
+pub unsafe extern "C" fn generate_identity(
+    wallet: *mut WalletFfi,
+    p: *mut PublicKey,
+    s: *mut PrivateKey,
```

Comment:
These are not required for this method as the Wallet will generate these keys internally as the name of the method implies.

Reply:
I did spend some time, the pr is coming shortly

Reply:
Not pr, additional commit